File: 3300-minimum-element-after-replacement-with-digit-sum/3300. Minimum Element After Replacement With Digit Sum.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        def convert(num):
            res = 0
            for i in str(num):
                res += int(i)
            return res
        res = 1e9
        for i in nums:
            res = min(res, convert(i))
        return res

File: 3300-minimum-element-after-replacement-with-digit-sum/3300. Minimum Element After Replacement With Digit Sum.java
public class Solution {
    public int minElement(int[] nums) {
        int res = Integer.MAX_VALUE;
        for (int n : nums) {
            int sum = 0;
            while (n > 0) {
                sum += n % 10;
                n /= 10;
            }
            res = Math.min(res, sum);
        }
        return res;
    }
}